api/src/test/java/org/pmiops/workbench/db/dao/DataSetServiceTest.java

```diff
@@ -488,6 +493,17 @@ public void testFITBITDomainToBigQueryConfig() {
         .contains("GROUP BY PERSON_ID, DATE");
   }
 
+  @Test
+  public void testDataDictionary() {
+    mockDataDictionaryDao();
```

Comment:
This Test is annotated with `@DataJpaTest` which means it's loading all the tables and using injected daos. Why are we using a mock call when you can query against the database? Also, I would add `@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)` just below `@DataJpaTest` so Spring will clean the context between test methods.

Reply:
Done